toml.md

```diff
@@ -405,9 +404,10 @@ apos15 = "Here are fifteen apostrophes: '''''''''''''''"
 str = ''''That,' she said, 'is still pointless.''''
 ```
 
-Control characters other than tab are not permitted in a literal string. Thus,
-for binary data, it is recommended that you use Base64 or another suitable ASCII
-or UTF-8 encoding. The handling of that encoding will be application-specific.
+Because most control characters are not permitted even in literal and multi-line
+literal strings, these literal strings are not suited for representing blobs of
+binary data.  It is recommended that you use Base64 or another suitable ASCII or
+UTF-8 encoding. The handling of that encoding will be application-specific.
```

Comment:
We have an alternative paragraph that expresses these same sentiments in #929.

Reply:
@eksortso, I believe the version of the paragraph in this PR is more explicit and direct with regards to literal and multi-line literal strings to the point than the one in #929.